cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/PushCommand.java

```diff
@@ -161,6 +162,10 @@ public void execute() {
                 if (balaPath == null) {
                     pushPackage(project);
                 } else {
+                    // If the file doesnt exist, and if the file is not a bala file
+                    if (!balaPath.toFile().exists() && !FileUtils.getExtension(balaPath).equals("bala")) {
```

Comment:
Shall we check the missing file and invalid extension cases separately and throw appropriate error messages?

Reply:
Checked the cases separately with the appropriate error messages